.github/workflows/scan-images.yaml

```diff
@@ -10,8 +10,8 @@ jobs:
     name: Scan images in bundle
     strategy:
       matrix:
-        release: [ 1.7 ]
-        risk: [ stable ]
+        # specfy location of bundle to be tested relative to releases/
```

Comment:
I think it makes more sense to specify the bundle location from the repo (inlcuding releases). This is a bit confusing

Reply:
OK.

Reply:
Actually, it is better to leave it this way. CI scripts takes input paramter release-risk. It will be very cumbersome to retrieve relase and risk from the marix value if we specify paths.

Reply:
Fixed.